csharp-refactoring/RefactoringChangePreventers/RefactoringChangePreventers/DivergentChange/Correct/Patient.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace RefactoringChangePreventers.DivergentChange.Correct;
+
+public class Patient
+{
+    public string FirstName { get; set; }
+    public string LastName { get; set; }
+    public string InsuranceNumber { get; set; }
+    public List<string> TreatmentHistory { get; set; }
+
+    public Patient(string firstName, string lastName, string insuranceNumber)
+    {
+        FirstName = firstName;
+        LastName = lastName;
+        InsuranceNumber = insuranceNumber;
+        TreatmentHistory = new List<string>();
+    }
+
+    public void AddTreatment(string treatment)
+    {
+        TreatmentHistory.Add(treatment);
+    }
+
+    public string GetPatientData()
+    {
+        return $"Name: {FirstName} {LastName}, Insurance Number: {InsuranceNumber}";
+    }
```

Comment:
I know this method was part of the example, but I don't think it's necessary. what's the difference between it and just having the properties? 
Let me know what you think if we just delete it and use the properties instead.

Reply:
Since, what's doing really is just a ToString alternative.